### ISSUE TITLE
loop in gettable error

### ISSUE THREAD

**Issue by james2doyle** [peersuasive/luce#5]
Trying both examples in the readme produces the following errors:

![](https://cl.ly/1N1W0q3e113o/Screen%20Shot%202016-08-24%20at%202.10.13%20PM.png)

You can see I am using the debug build on OSX

**Comment by peersuasive** [comment]
Yes, I know this error. There's an infinite loop in some occasions in the messy LClass -- but I may have fixed it with v0.3.4, though, as it doesn't occur on Linux anymore.
